src/python/dxpy/cli/output_handling.py

```diff
@@ -140,7 +144,8 @@ def write_expression_output(
             "delimiter": DELIMITER,
             "lineterminator": OS_SPECIFIC_LINE_SEPARATOR,
             "quoting": QUOTING,
-            "quotechar": QUOTE_CHAR,
+            "quotechar": QUOTE_CHAR,\
+            "restval":"None"
```

Comment:
Might need to check with PL, but I think it's fine if cell is empty, so `None` instead of `"None"`

```suggestion
            "restval":"None"
```